presidio-anonymizer/README.MD

```diff
@@ -1 +1,121 @@
-# TODO add the document Omri wrote.
```

Comment:
add task/user story here for the TODO

Reply:
This is no longer relevant. I did for others.